### DISCUSSION TITLE
Try the new Single Project MSIX support! 📦

### DISCUSSION THREAD

**Discussion by andrewleader** [microsoft/WindowsAppSDK#1283] [Announcements]
We've just released experimental support of Single Project MSIX - no more packaging project for most basic apps! This means you can have a packaged desktop app that is just one single project inside Visual Studio, rather than two!

![image](https://user-images.githubusercontent.com/13246069/130292059-bc6b0989-b435-4b5c-bc2d-80e55694e9cf.png)

You can try it out here: https://docs.microsoft.com/windows/apps/windows-app-sdk/single-project-msix

Please let us know how it went and share your feedback! This will be incorporated with the WinUI 3 project templates in an upcoming preview release, so you won't have to do all of these manual conversion steps in the future.

**Comment by shelllet** [comment]
many compile errors! what can i do?

![image](https://user-images.githubusercontent.com/16729017/131605492-39744b06-b8d0-49ed-a323-e4faf5d6e3ae.png)

**Reply by andrewleader** [reply]
@alwu-msft can you help with these errors shelllet is facing when trying to use Single Project MSIX?

**Reply by shelllet** [reply]
All thease errors seem to  point to one same problem , `win32 api`  can't  found.  even I already include `windows.h` in 'pch.h' .
